packages/state-manager/src/sagas/messages/verifyMessage/verifyMessages.saga.ts

```diff
@@ -1,43 +1,26 @@
 import { PayloadAction } from '@reduxjs/toolkit'
 import { select, call, put, spawn } from 'typed-redux-saga'
-import { getCrypto } from 'pkijs'
-import { messagesSelectors } from '../messages.selectors'
-import { stringToArrayBuffer } from 'pvutils'
-import { keyObjectFromString, verifySignature } from '@quiet/identity'
 import { messagesActions } from '../messages.slice'
 import { MessageVerificationStatus } from '../messages.types'
 import { ChannelMessage } from '../../publicChannels/publicChannels.types'
-import { publicChannelsActions } from '../../publicChannels/publicChannels.slice'
+import { messagesSelectors } from '../messages.selectors'
 
 export function* verifyMessagesSaga(
-  action: PayloadAction<ReturnType<typeof publicChannelsActions.cacheMessages>>['payload']
+  action: PayloadAction<ReturnType<typeof messagesActions.incomingMessages>>['payload']
 ): Generator {
-  const crypto = getCrypto()
   const messages = action.payload.messages
+  const verifiedStatus = action.payload.verifiedStatus
+
   for (const message of messages) {
-    yield* spawn(verifyMessage, message, crypto)
+    yield* spawn(verifyMessage, message, verifiedStatus)
   }
 }
 
-function* verifyMessage(message: ChannelMessage, crypto: SubtleCrypto): Generator {
-  const signature = stringToArrayBuffer(message.signature)
-
-  const publicKeysMapping = yield* select(messagesSelectors.publicKeysMapping)
-
-  let cryptoKey = publicKeysMapping[message.pubKey]
-  if (cryptoKey === undefined) {
-    cryptoKey = yield* call(keyObjectFromString, message.pubKey, crypto)
-    yield* put(
-      messagesActions.addPublicKeyMapping({ publicKey: message.pubKey, cryptoKey: cryptoKey })
-    )
-  }
-
-  const verified = yield* call(verifySignature, signature, message.message, cryptoKey)
-
+function* verifyMessage(message: ChannelMessage, verifiedStatus: boolean): Generator {
```

Comment:
1. This function doesn't do any verification, so why is it called verify....?
2. It's redundant, because it's only puts actions